tests/infra/path.py

```diff
@@ -39,10 +39,7 @@ def build_lib_path(lib_name, enclave_type=None, library_dir="."):
         return os.path.join(library_dir, os.path.normpath(f"{lib_name}{ext}"))
 
 
-def build_bin_path(bin_name, enclave_type=None, binary_dir="."):
-    if enclave_type == "virtual":
-        bin_name = f"{bin_name}.virtual"
-
+def build_bin_path(bin_name, binary_dir="."):
```

Comment:
I believe this is what breaks the LTS compatibility test for `NoSGX`: 1.x nodes still need to start with `cchost.virtual`.

Reply:
I assumed so, but unable to repro locally. Will investigate when I get time.